src/proof_system/preprocessed_circuit.rs

```diff
@@ -46,7 +48,7 @@ impl PreProcessedCircuit {
         transcript.append_commitment(b"q_arith", &self.arithmetic.q_arith.commitment);
         transcript.append_commitment(b"q_range", &self.range.q_range.commitment);
         transcript.append_commitment(b"q_logic", &self.logic.q_logic.commitment);
-
+        //transcript.append_commitment(b"q_ecc", &self.ecc.q_ecc.commitment);
```

Comment:
Why is ecc_gate commented out? The whole point is to have it activated to be able to test it.